src/media/js/addon/components/addonSubnav.js

```diff
@@ -8,9 +8,9 @@ export default class AddonSubnav extends React.Component {
   render() {
     return (
       <Subnav>
-        <ReverseLink to="addon-dashboard">My Firefox OS Add-ons</ReverseLink>
```

Comment:
we want them displayed publicly as Firefox OS Add-ons. It'll prevents confusion and seem fresher.